### ISSUE TITLE
Try on different beard styles for men for fun or to check which suits them the most {SSoC 2.0}

### ISSUE THREAD

**Issue by HadiaSamreen** [akshitagupta15june/Face-X#1529]
if someone is interested to join me(after the issue gets assigned),you are most welcome!

**Comment by impeccable16** [comment]
hi I am interested

**Comment by HadiaSamreen** [comment]
Hi @impeccable16 ! You are most welcome! Let's connect (after the issue gets assigned) to discuss further and work on it.

**Comment by HadiaSamreen** [comment]
@akshitagupta15june

**Comment by HadiaSamreen** [comment]
@akshitagupta15june

**Comment by impeccable16** [comment]
Sure! @HadiaSamreen

**Comment by akshitagupta15june** [comment]
Yeah go for it
But add in snapchat folder

**Comment by HadiaSamreen** [comment]
Sure! Thank you

**Comment by HadiaSamreen** [comment]
Yes @impeccable16 ...kindly share your contact details on my email i'd like whatsapp number or something through which we could connect and collaborate

**Comment by impeccable16** [comment]
Hi, @HadiaSamreen  can you please share your email id so that I can mail you my contact number.

**Comment by HadiaSamreen** [comment]
@impeccable16 huhdia0905@gmail.com

**Comment by impeccable16** [comment]
@HadiaSamreen  I have sent you the email.
